src/api/api.types.tsx

```diff
@@ -0,0 +1,46 @@
+interface CreatedModifiedMixin {
+  created_time: string;
+  modified_time: string;
+}
+
+export interface APIError {
+  detail: string;
+}
+
+// Manufacturers
```

Comment:
Could you make this a bit clearer for those scrolling through perhaps like in the MSW handlers:
`// ------------------------------------ MANUFACTURERS ------------------------------------`
Happy for this to be done further up the chain.